tests/unittests/CoreText/CTFontTests.mm

```diff
@@ -73,8 +72,7 @@ virtual void TearDown() {
 
 INSTANTIATE_TEST_CASE_P(CTFont,
                         FontCopyName,
-                        ::testing::Values(::testing::make_tuple(kCTFontCopyrightNameKey, c_copyrightName),
```

Comment:
confused why you took this approach - why not just swap out the constants above for the ones in WinObjC.ttf, and reuse this parameterized test instead of adding a new one?